project_earned_value_task/data/project_data.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="0">
+
+        <!-- Task Stages -->
+        <record id="project.project_tt_analysis" model="project.task.type">
+            <field name="poc">0</field>
+        </record>
+        <record id="project.project_tt_specification"
+                model="project.task.type">
+            <field name="poc">0</field>
+        </record>
+        <record id="project.project_tt_design" model="project.task.type">
+            <field name="poc">0</field>
+        </record>
+        <record id="project.project_tt_development" model="project.task.type">
+            <field name="poc">0</field>
+        </record>
+        <record id="project.project_tt_testing" model="project.task.type">
+            <field name="poc">0</field>
+        </record>
+        <record id="project.project_tt_merge" model="project.task.type">
+            <field name="poc">0</field>
+        </record>
+        <record id="project.project_tt_deployment" model="project.task.type">
+            <field name="poc">100</field>
+        </record>
+        <record id="project.project_tt_cancel" model="project.task.type">
+            <field name="poc">0</field>
+        </record>
+    </data>
+
+</openerp>
```

Comment:
This is just a proposal of the percentage completed for the task stages that are predefined in the system. If we do not set it, the POC will default to 0 for all stages as a default.